modules/control/controller/lat_controller.cc

```diff
@@ -86,7 +86,9 @@ LatController::LatController() : name_("LQR-based Lateral Controller") {
   AINFO << "Using " << name_;
 }
 
-LatController::~LatController() { CloseLogFile(); }
+LatController::~LatController() {
+  CloseLogFile();
+}
```

Comment:
These are auto-adjusted by clang-format tool. No need to change.

Reply:
Fixed. Thanks.